ddl/foreign_key_test.go

```diff
@@ -196,7 +198,7 @@ func (s *testForeighKeySuite) TestForeignKey(c *C) {
 		}
 		checkOK = true
 	}
-
+	d.SetHook(tc)
```

Comment:
`tc` should be a new hook variable before #line 182.

Reply:
I guess I know what I should do now.
Thanks,bro!